exonum/src/storage/proof_map_index/mod.rs

```diff
@@ -1057,13 +1057,75 @@ where
     }
 }
 
-impl<T, K, V> ::std::fmt::Debug for ProofMapIndex<T, K, V>
+impl<T, K, V> fmt::Debug for ProofMapIndex<T, K, V>
```

Comment:
I've refactored the `Debug` implementation for `ProofMapIndex` in order not to use `Box`es. IMO, this one is more understandable as well.